lib/actions/enter.coffee

```diff
@@ -0,0 +1,75 @@
+###
+Copyright 2016 Resin.io
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+###
+
+getSubShellCommand = (command) ->
```

Comment:
That sounds like a good idea! By the way I basically used the code from here: https://github.com/resin-io-modules/resin-sync/blob/master/lib/shell.coffee . So yeah, instead of c/p it would be nice to have this as a separate module. If you feel that this is not urgent, please create a ticket and assign it to me so I can create the module later during this week!